nncf/tensorflow/quantization/algorithm.py

```diff
@@ -263,6 +264,12 @@ def _parse_range_init_params(self) -> TFRangeInitParams:
     def _parse_group_params(self, quant_config: Dict, quantizer_group: QuantizerGroup) -> None:
         group_name = quantizer_group.value
         params_dict = quant_config.get(group_name, {})
+        q_mode = params_dict.get('mode')
+        if q_mode is not None and 'preset' in quant_config:
+            raise RuntimeError("Global constraints cannot be parsed. Only quantize mode or preset must be set.")
```

Comment:
I would expect the user to be able to override some of the preset options via config and not get a RuntimeError. I mean, If we will have a more complex preset which will be had more parameters not only mode. Should we raise a runtime error for all parameters in the preset?